README.md

```diff
@@ -1,6 +1,9 @@
 HaikuTwitter
 ============
 
+![HaikuTwitter screenshot](HaikuTwitter.png "HaikuTwitter")
```

Comment:
Can you take a more interesting screenshot, showing twittering in action?
Move the screenshot below the intro sentence "A native Twitter client for Haiku."

Reply:
OK, waddlesplach informed me that Twitter doesn't exactly work at the moment...
So, keep the screenshot then, just move it after the intro sentence.